### ISSUE TITLE
Autosave is Still a Huge Issue

### ISSUE THREAD

**Issue by KalaMiDeviL** [sp614x/optifine#1758]
With the mods I want and my pc specs, it is impossible to play the game with constant autosaving, so thanks for your efforts. However, with the 30min intervall I still get a recurring 10s game freeze that I can't see coming. This makes hardcore mode absolutely unplayable. Please add one of the following:
- a notification, that the game is about to save 10s or 20s before, so I can dig in, tower up or make a temporary wall
- the option to completely turn off autosave off

Please please please!

**Comment by Regnander** [comment]
Regarding the second option, how about saving when the pause screen/game menu is opened, so the game at least can save your progress?

**Comment by KalaMiDeviL** [comment]
I am not sure what you mean. Does the interval get reset if I save myself? And if yes, how do I save without going back to the main menu?

**Comment by Regnander** [comment]
No, no. I suggested that the game could save when the player pauses it (by pressing ESC in-game) if sp614x decided to add an option that allowed players to disable autosaving.

**Comment by KalaMiDeviL** [comment]
Oh, that is a great idea!
